frontend/test/metabase/scenarios/question/nested.cy.spec.js

```diff
@@ -553,26 +555,32 @@ describe("scenarios > question > nested", () => {
 });
 
 function createNestedQuestion(
-  { baseQuestionDetails, nestedQuestionDetails },
+  { baseQuestionDetails, nestedQuestionDetails = {} },
   { loadBaseQuestionMetadata = false, visitNestedQuestion = true } = {},
 ) {
+  if (!baseQuestionDetails) {
+    throw new Error("Please provide the base question details");
+  }
+
   createBaseQuestion(baseQuestionDetails).then(({ body: { id } }) => {
     loadBaseQuestionMetadata && visitQuestion(id);
 
-    return cy.createQuestion(
-      {
-        name: "Nested Question",
-        query: {
-          "source-table": `card__${id}`,
-        },
-        ...nestedQuestionDetails,
-      },
-      {
-        visitQuestion: visitNestedQuestion,
-        wrapId: true,
-        idAlias: "nestedQuestionId",
+    const { query: nestedQuery, ...details } = nestedQuestionDetails;
```

Comment:
We need to omit the query, or to extract the "rest" of the details from `nestedQuestionDetails` in order to preserve the default query with the `"source-table"` in the `composite` object.